.github/workflows/conformance-ingress.yaml

```diff
@@ -95,7 +95,6 @@ jobs:
             --helm-set=ingressController.loadbalancerMode=${{ matrix.loadbalancer-mode }} \
             --helm-set=ingressController.default=${{ matrix.default-ingress-controller }} \
             --helm-set=extraConfig.bpf-lb-acceleration=${{ matrix.bpf-lb-acceleration }} \
-            --rollback=false \
             --version="
```

Comment:
can you remove the version flag as well ? I think this is legacy flag before helm template/install mode is added.